fe1-web/model/objects/WalletCryptographyHandler.ts

```diff
@@ -33,11 +32,19 @@ export class WalletCryptographyHandler {
   /* usages for the RSA key */
   private readonly keyUsages: KeyUsage[] = ['encrypt', 'decrypt'];
 
-  /* the crypto library is passed to constructor, this is necessary in order to test the
-     cryptography handler without the crypto.subtle library provided by the window object.
-     In jest context provide a MOCK crypto.subtle library, otherwise provide window.crypto */
-  constructor(cryptography: Crypto) {
-    this.cryptography = cryptography;
+  /**
+   * the crypto library is passed to constructor, this is necessary in order to test the
+   * cryptography handler without the crypto.subtle library provided by the window object.
+   * In jest context provide a MOCK crypto.subtle library, otherwise window.crypto is
+   * selected by default.
+   * @param cryptography MOCK crypto.subtle library or nothing (window.crypto default)
+   */
+  constructor(cryptography?: Crypto) {
+    if (!cryptography) {
+      this.cryptography = window.crypto;
```

Comment:
This will not work in a react-native context.
If you accept a Crypto interface, whoever calls this can pass on a polyfill for react-native.
On the other hand, given that it's awkward to pass the crypto library around, you might want to have a separate file allowing you to obtain crypto (e.g. `getCrypto(): Crypto`) , which returns window.crypto in the browser, something else in react-native, and which can be mocked during tests.